src/components/Comments.jsx

```diff
@@ -19,13 +23,66 @@ const Comments = ({ article_id, comments, setComments }) => {
       });
   }, [article_id, setComments]);
 
+  const handleCommentSubmit = (event) => {
+    event.preventDefault();
+    setIsLoading(true);
+    const newComment = {
+      article_id: article_id,
```

Comment:
article_id is needed for the body of this request; it's just used in the endpoint